apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -124,6 +127,7 @@ function ListBoxInline({ options, layout }) {
   if (layout?.toolbar !== undefined) {
     toolbar = layout.toolbar;
   }
+  const toolbarDetachedOnly = toolbar && layout?.title === '';
```

Comment:
Don't if title can be undefined… but:
```suggestion
  const toolbarDetachedOnly = toolbar && !layout?.title?.length
```

Reply:
I don't either, but it should only be detached if you explicitly set it to an empty string.

Reply:
I was just thinking of the case when an undefined value slips in here, should this be `true` or `false` then?